src/components/Avatar/Avatar.types.ts

```diff
@@ -60,3 +62,67 @@ export interface AvatarProps
      */
     alt?: string;
 }
+
+interface MaxAvatarProps extends BaseAvatarProps {
+    /**
+     * Max Avatar class names.
+     */
+    classNames?: string;
+    /**
+     * Shown Avatars max count.
+     */
+    count?: number;
+    /**
+     * Max Avatar style.
+     */
+    style?: React.CSSProperties;
+    /**
+     * Max tooltip props.
+     */
+    tooltipProps?: TooltipProps;
+    /**
+     * Max Avatar custom value.
+     */
+    value?: ReactNode;
+    /**
+     * Unique id used to target element for testing.
+     */
+    'data-test-id'?: string;
+    /**
+     * Max Avatar ref.
+     */
+    ref?: Ref<HTMLDivElement>;
+}
+
+interface AvatarListProps
+    extends Omit<ListProps<ReactNode>, 'footer' | 'header' | 'layout'> {}
+
+export interface AvatarGroupProps extends OcBaseProps<HTMLDivElement> {
+    /**
+     * Avatar group List props.
+     */
+    avatarListProps?: AvatarListProps;
+    /**
+     * Avatar group children.
+     */
+    children?: ReactNode;
+    /**
+     * Avatar group fallback font size
+     * @default 18
+     */
+    fontSize?: string;
+    /**
+     * Avatar group max props.
+     */
+    maxProps?: MaxAvatarProps;
+    /**
+     * Avatar group size.
```

Comment:
presuming this is the size of a single avatar within the group, rather than the group size

Reply:
Yes.